generic.go

```diff
@@ -110,6 +110,28 @@ func (api API) merge(path string, input interface{}) error {
 	return wrapErrors(res, []byte(body), errs)
 }
 
+// UPDATE
+func (api API) unmerge(path string, input interface{}) error {
```

Comment:
[blocking] it's not necessary to add a complete duplicate of the `merge` method. Don't duplicate the code, instead find a good name that fits both. If need be, alias the method.

The `merge` method is also very similar to `create`, the only difference is lack of `EndStruct(output)`, but I suggest not to tackle that now.